internal/test/test.go

```diff
@@ -19,10 +19,9 @@ package test
 import (
 	"bytes"
 	"flag"
+	"github.com/pkg/errors"
```

Comment:
this is a formatting thing of go vet, should this be commited?

Reply:
formatting indeed,this is `make test` autoformatting it via `go vet`

I think we commit it, otherwise its gonna end up there sooner or later because we will always run make test before committing....

Reply:
Ok